components/BottomNavigation/examples/BottomNavigationBlurExample.m

```diff
@@ -35,10 +33,11 @@ @implementation BottomNavigationBlurExample
 - (id)init {
   self = [super init];
   if (self) {
-    _colorScheme =
+    MDCContainerScheme *containerScheme = [[MDCContainerScheme alloc] init];
+    containerScheme.colorScheme =
         [[MDCSemanticColorScheme alloc] initWithDefaults:MDCColorSchemeDefaultsMaterial201804];
-    _typographyScheme =
-        [[MDCTypographyScheme alloc] initWithDefaults:MDCTypographySchemeDefaultsMaterial201804];
+    containerScheme.typographyScheme = [[MDCTypographyScheme alloc] init];
```

Comment:
Tip: I'd prefer to keep the default values in the container scheme. That would match most of our examples in the catalog.

Reply:
Done.